README.md

```diff
@@ -101,14 +132,102 @@ credentials, ie:
 }
 ```
 
-## Publishing
+The contents of this file can be found in lastpass with the name
```

Comment:
The user should use what's in lastpass.  If the entry gets deleted the can search through the providers docs on how to create credentials for publishing on the respective clouds.